### PR TITLE
Translate hardhat page updates, PR 785/806

### PR DESCRIPTION
**Pull request by wuzhong-papermoon**
### Description/Original PRs
https://github.com/moonbeam-foundation/moonbeam-docs/pull/785
https://github.com/moonbeam-foundation/moonbeam-docs/pull/806/files

### CODE REVIEW

**Review by wuzhong-papermoon — COMMENTED** [review]
Some typo fixed.

File: builders/build/eth-api/dev-env/hardhat.md [hunk 1]
```diff
@@ -52,135 +58,172 @@ description: 了解如何使用Hardhat在Moonbeam上编译、部署和调试以
...
+          url: '{{ networks.moonbase.rpc_url }}', // 输入您的RPC URL
+          chainId: {{ networks.moonbase.chain_id }}, // (hex: {{ networks.moonbase.hex_chain_id }}),
+          accounts: ['INSERT_PRIVATE_KEY'],
+        },
+      },
+    };
+    ```
 
-    === "Moonbeam开发节点"
+=== "Moonbeam 开发者节点"
```
Comment (eshaben) [inline builders/build/eth-api/dev-env/hardhat.md:133]: For all of the other instances throughout the docs, we use `"Moonbeam开发节点"`. If you think we should change it to `"Moonbeam 开发者节点"`, then we should probably change it everywhere else.

Whats the difference between the two? Do they both say "Moonbeam Development Node"?

File: builders/build/eth-api/dev-env/hardhat.md [hunk 2]
```diff
@@ -234,13 +280,14 @@ main()
   });
 ```
 
-您现在可以使用`run`命令并指定`moonbase`作为网络来部署`Box.sol`合约：
+使用以下命令来运行脚本，部署`Box.sol`合约；您需要给这个命令定义一个部署合约的网络名称，这个名称已在`hardhat.config.js`中定义。
```
Comment (eshaben) [inline builders/build/eth-api/dev-env/hardhat.md:283]: In the above code snippet, the comments were changed from `// 2. Instantiating a new Box smart contract` to `// 2. Instantiate a new Box smart contract`

And similarly, `  // 3. Waiting for the deployment to resolve` was changed to `  // 3. Wait for the deployment to resolve`

Some of the comments in the other snippets were translated to CN. I feel like if we are doing some snippets, we should just do all of them so we are being consistent

File: builders/build/eth-api/dev-env/hardhat.md [hunk 3]
```diff
@@ -7,15 +7,21 @@ description: 了解如何使用Hardhat在Moonbeam上编译、部署和调试以
 
 ## 概览 {: #introduction }
 
-[Hardhat](https://hardhat.org/){target=_blank}是一个以太坊开发环境，可帮助开发人员管理和自动化构建智能合约和DApp所固有的重复性任务。Hardhat可以直接与Moonbeam的以太坊API交互，因此可以在部署智能合约至Moonbeam时使用。
+[Hardhat](https://hardhat.org/){target=_blank}是一个灵活可拓展的以太坊开发环境，它能帮助开发者简化智能合约开发过程。 由于Moonbeam与以太坊兼容，您可以在Moonbeam上使用Hardhat开发和部署智能合约。
```
Comment (eshaben) [inline builders/build/eth-api/dev-env/hardhat.md:10]: can you please update this entire page so all of the `target=_blank`s on this page are using `target=\_blank` instead?

**Review by eshaben — CHANGES_REQUESTED** [review]
Thank you! Looks good! Just a few things need to be fixed 🙂

**Comment by eshaben** [comment]
@wuzhong-papermoon there are some merge conflicts too, can you please resolve those? if you have any questions or want to walk through that process together, let me know!

File: builders/build/eth-api/dev-env/hardhat.md (hunk 1 above)
Comment (wuzhong-papermoon) [reply builders/build/eth-api/dev-env/hardhat.md:133]: They essential mean the same thing, thanks for pointing it out! The original version is more accurate actually. 
```suggestion
=== "Moonbeam 开发节点"
```

Comment (eshaben) [reply builders/build/eth-api/dev-env/hardhat.md:133]: Ok, thanks for looking into it. I'll leave it as is then

**Comment by eshaben** [comment]
@wuzhong-papermoon more changes were made to the hardhat pages in [this PR](https://github.com/moonbeam-foundation/moonbeam-docs/pull/902/files#diff-535760d7c6a913604bf97c1f222fe2bdd985854b8a4e95e53203ff8387f84cb8)

It might be good to just do a quick scan over the page to make sure we didn't forget anything/in case anything else needs to be updated

File: builders/build/eth-api/dev-env/hardhat.md (hunk 2 above)
Comment (wuzhong-papermoon) [reply builders/build/eth-api/dev-env/hardhat.md]: Done!

**Comment by wuzhong-papermoon** [comment]
Hey @eshaben , I pushed a new commit with hardhat ignition contents. Would you take a look please!

**Review by eshaben — COMMENTED** [review]
looks good. just one thing!

**Review by eshaben — APPROVED** [review]
LGTM!

**Pull request merged**